mlflow/models/evaluation/base.py

```diff
@@ -320,6 +320,14 @@ def _gen_md5_for_arraylike_obj(md5_gen, data):
                 )
             )
 
+    @property
+    def feature_names(self):
+        import numpy as np
+        if isinstance(self.data, np.ndarray):
+            return [f'f{i}' for i in range(self.data.shape[1])]
+        else:
+            return [c for c in self.data.columns if c != self.labels]
```

Comment:
Model explainability display require display feature names.
Shall we force user set meaningful feature names for "ndarray" case data ?

Reply:
I don't think we need to force users to set their own feature names for ndarray, though it would be nice to give users an optional `feature_names` argument for customizing the feature names. Thanks Weichen!